docs/config.md

```diff
@@ -415,6 +415,7 @@ retentions = 1s:35d:10min:7
 # * aggregationMethod specifies the functions used to aggregate values for the next retention level. Legal methods are avg/average, sum, min, max, and last. The default is average.
 # Unlike Graphite, you can specify multiple, as it is often handy to have different summaries available depending on what analysis you need to do.
 # When using multiple, the first one is used for reading.  In the future, we will add capabilities to select the different archives for reading.
+# * reorderBuffer an optional buffer that temporarily keeps data points in memory as raw data and allows insertion at random order. The specified value is how many datapoints, at a defined metric interval, should be kept before they are flushed out. This is useful if the metric producers cannot guarantee that the data will arrive in order, but it is relatively memory intensive.
```

Comment:
"at a defined interval" is vague. should probably be "based on the specified raw interval"